kano/utils/hardware.py

```diff
@@ -19,18 +29,83 @@
 RPI_COMPUTE_SCORE = 4000
 RPI_2_B_SCORE = 5000
 
-# "performance" scores lookup table with keys as given by get_rpi_model()
-PERFORMANCE_SCORES = {
-    'RPI/A': RPI_A_SCORE,
-    'RPI/A+': RPI_A_PLUS_SCORE,
-    'RPI/B': RPI_B_SCORE,
-    'RPI/B+': RPI_B_PLUS_SCORE,
-    'RPI/Zero': RPI_ZERO_SCORE,
-    'RPI/Compute': RPI_COMPUTE_SCORE,
-    'RPI/2/B': RPI_2_B_SCORE,
+RPI_1_CPU_PROFILE = 'rpi_1'
+RPI_2_CPU_PROFILE = 'rpi_2'
+
+
+'''
+Lookup table with keys as given by get_rpi_model() containing:
+    * Human readable 'name' of the board
+    * 'cpu_profile' which details the settings to use
+    * 'performance' scores
+'''
+BOARD_PROPERTIES = {
```

Comment:
@radujipa I absolutely agree but this looks like it could be a large task. We'll keep it like this for now with the goal to merge them at some point.